coalib/bears/Bear.py

```diff
@@ -296,27 +308,14 @@ def missing_dependencies(cls, lst):
                     instances).
         :return:    A list of missing dependencies.
         """
-        dep_classes = cls.get_dependencies()
+        dep_classes = cls.BEAR_DEPS
```

Comment:
here you're assigning a list of strings to a variable to suggest to hold class objects, that can never work